packages/fast-components-react-base/src/context-menu/context-menu.props.ts

```diff
@@ -12,6 +12,11 @@ export interface ContextMenuHandledProps extends ContextMenuManagedClasses {
      * The context menu children
      */
     children?: React.ReactNode;
+
+    /**
+     * When true, the context menu will immediately place focus on the appropriate menu-item when the context-menu mounts

```

Comment:
nit: if you update this, can you remove the hyphens between menu item and content menu, so it is consistent with the way we write it elsewhere?